Source/Flow/Private/Nodes/Route/FlowNode_NamedRerouteUsage.cpp

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "Nodes/Route/FlowNode_NamedRerouteUsage.h"
+
+#include "FlowAsset.h"
+
+UFlowNode_NamedRerouteUsage::UFlowNode_NamedRerouteUsage(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+#if WITH_EDITOR
+	Category = TEXT("Named Reroute");
+	NodeStyle = EFlowNodeStyle::Custom;
+#endif
+	InputPins = {};
+	OutputPins = {FFlowPin(TEXT("Out"))};
+	AllowedSignalModes = {EFlowSignalMode::Enabled, EFlowSignalMode::Disabled, EFlowSignalMode::PassThrough};
+}
+
+FText UFlowNode_NamedRerouteUsage::GetNodeTitle() const
+{
+	return FText::FromString(NodeTitle.ToString());
+}
+
+void UFlowNode_NamedRerouteUsage::SetNodeName()
+{
+	if (Declaration)
+	{
+		NodeTitle = FName(*Declaration->GetNodeTitle().ToString());
+	}
+	else
+	{
+		NodeTitle = FName(TEXT("Invalid named reroute"));
+	}
+}
+
+#if WITH_EDITOR
+void UFlowNode_NamedRerouteUsage::PostEditChangeProperty(FPropertyChangedEvent& PropertyChangedEvent)
```

Comment:
Looks like redundant function to me, neither Declaration, nor NodeTitle can be modified by user